### DISCUSSION TITLE
Install-Package: Dependency loop detected for package 'Betalgo.OpenAI.GPT3'

### DISCUSSION THREAD

**Discussion by InTEGr8or** [betalgo/openai#70] [Q&A]
Getting this error when installing

```ps1
Install-Package Betalgo.OpenAI.GPT3

The package(s) come(s) from a package source that is not marked as trusted.
Are you sure you want to install software from 'nuget.org'?
[Y] Yes  [A] Yes to All  [N] No  [L] No to All  [S] Suspend  [?] Help (default is "N"): A
Install-Package: Dependency loop detected for package 'Betalgo.OpenAI.GPT3'.
```

**Result**
It runs really slow, and then fails.

**Expected behavior**
Install the package from the CLI

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Language: Powershell Core 7.3.1
 - Version: 7.3.1

**Comment by kayhantolga** [comment]
I am not sure what is the reason you got such as error. I have never seen it before. Do you have any idea what might be causing it?

I found a couple of articles that suggest skipping the dependency check and updating the NuGet client version. This might help you but I am not sure it is the best solution. 
https://medium.com/@mike_82447/nuget-package-management-with-powershell-180626ebe14c

As far as I understand this is most likely a NuGet client issue rather than an `openai.gpt3` issue. So I will move this item to discussions.
